eslintrc.js

```diff
@@ -20,12 +20,34 @@ module.exports = {
     "sort-requires"
   ],
   "rules": {
+    "array-callback-return": "error",
+    "dot-notation": "error",
     "eqeqeq": "error",
```

Comment:
How about setting the [`smart`](https://eslint.org/docs/rules/eqeqeq#smart) option? See also #69

```suggestion
     "eqeqeq": ["error", "smart"],
```

Reply:
I personally haven't used this rule in production so I don't know how many errors we'll get. It would be better to enable this in the main repo in a test branch first and see the results.

Reply:
When I tried the option to stylelint/stylelint master branch, the following output is got:

```
lib/rules/function-calc-no-invalid/index.js
  158:30  error  'eqeqeq' rule is disabled but never reported  eslint-comments/no-unused-disable
```

https://github.com/stylelint/stylelint/blob/90bef807ac270ae4082384de71ada7f56bbb3145/lib/rules/function-calc-no-invalid/index.js#L158-L161

Reply:
That's OK, it is a suppression that doesn't apply anymore.